rocketpy/EnvironmentAnalysis.py

```diff
@@ -236,15 +237,19 @@ def __check_extra_requirements(self):
         env_analysis_require = [
             "timezonefinder",
             "windrose>=1.6.8",
+            "IPython>=8.8.0",
             "ipywidgets>=7.6.3",
             "jsonpickle",
         ]
```

Comment:
If the package name is splitted and only the first part of the name matters for checking if it is present, why put the version numbers in this list?

I might have overlooked something, but it seems to me that 

```py
        env_analysis_require = [
            "timezonefinder",
            "windrose",
            "IPython",
            "ipywidgets",
            "jsonpickle",
        ]
```

would have the same effect for this function without the need of the versioning. Perhaps it would be useful to keep the versioning and also check if the installed version is compliant, it is likely that the `importlib` has ways to check that.

Reply:
Yes I had left the numbers bc I thought in the future it would be nice to have version numbering tests.
I just "hadn't realized by my own" that this future were so close, so I implemented version checks in my last commit. 
This feature can be used in other classes in the future.

Hope you like it now!